test/ios_application_test.sh

```diff
@@ -586,6 +586,46 @@ function test_prebuilt_static_apple_static_framework_import_resources() {
       "Payload/app.app/fmwk.bundle/Some.plist"
 }
 
+# Tests that a dynamic framework with a "." in the name builds
+function test_ios_framework_name_with_dot() {
+  create_common_files
+  cat >> app/BUILD <<EOF
+load("@build_bazel_rules_apple//apple:ios.bzl",
+    "ios_framework"
+)
+ios_framework(
+    name = "fmwk.dynamic_framework",
```

Comment:
instead of adding to the shell tests which ideally we would get rid of can you add this under the targets_to_test BUILD files

Reply:
Added an `archive_contents_test`